test/expint.jl

```diff
@@ -46,7 +46,7 @@ using Base.MathConstants
 
     @testset "En" begin
         # ≅ but worse
-        ⩭(x, y) = relerrc(x, y) ≤ 5e-12
+        ⩭(x, y) = relerr(x, y) ≤ 5e-12
```

Comment:
Why is this changed?  You are making the tests weaker by changing `relerrc` to `relerr`.

Reply:
Actually I didn't want to weaken the tests.
`relerrc` can be expressed with `relerr` using dynamic dispatch and this is what I was trying to do.
The test cases failed because I messed up locally by not using a fresh REPL.
I push a correct version of `relerr` for complex input